.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: run-tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
```

Comment:
Maybe add a [matrix](https://github.com/getsidekicker/laravel-event-bridge/blob/main/.github/workflows/run-tests.yml#L10-L19) for different PHP and Laravel versions.

Reply:
I might skip this for now because it requires a fairly complex setup given it's using docker-compose + multiple containers